packages/SwingSet/package.json

```diff
@@ -12,9 +12,10 @@
   },
   "scripts": {
     "build": "exit 0",
-    "test": "ava",
```

Comment:
instead of changing `test`, you may should add a separate `test:coverage` script.

Reply:
nvm, just noticed `coverage`. You can revert this line then?